src/core/utils.go

```diff
@@ -290,6 +290,11 @@ func ExecWithTimeout(target *BuildTarget, dir string, env []string, timeout time
 		cmd.Stdout = io.MultiWriter(&out, &outerr)
 		cmd.Stderr = &outerr
 	}
+	if attachStdin {
```

Comment:
Thoughts on a variable name that could be taken to mean all 3, since all 3 get attached? E.g., `attachStdStreams`?